embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/EmbraceExtensions.kt

```diff
@@ -121,6 +121,33 @@ internal fun Span.setSequenceId(id: Long): Span {
     return this
 }
 
+/**
+ * Add in interesting attributes about the running app environment to the span as private Embrace attributes
+ */
+internal fun Span.addAppAttributes(): Span {
+    var hasOkHttp3 = false
+    var okhttpVersion: String? = null
+    try {
+        val okhttp = Class.forName("okhttp3.OkHttp", false, javaClass.classLoader)
+        hasOkHttp3 = true
+        okhttpVersion = okhttp.getField("VERSION").get(okhttp)?.toString()
+    } catch (t: Throwable) {
+        // only write the version if the call to obtain it doesn't throw
+    }
+    setAttribute("okhttp3".toEmbraceAttributeName(), hasOkHttp3.toString())
+    okhttpVersion?.let { setAttribute("okhttp3_version".toEmbraceAttributeName(), it) }
+
+    var kotlinStdLibVersion = "0.0.0"
```

Comment:
nit: would 'Unknown' or something similar be a more explicit way of stating we couldn't get the version?

Reply:
Yeah it's a string so this shouldn't break any filtering. I'll do that.